lib/commands/setViewportSize.js

```diff
@@ -37,16 +36,19 @@ module.exports = function setViewportSize(size) {
         throw new ErrorHandler.CommandError('number or type of arguments don\'t agree with setViewportSize command');
     }
 
-    return setSize(this);
+    var shouldIndent = typeof(type) === 'undefined' ? true : type;
```

Comment:
According to our docs the default value of type is false (apply windowHandleSize), so we should switch `true` and `type`.